_includes/footer.html

```diff
@@ -8,11 +8,11 @@
         </ul>
       </div>
       <div class="col-md-6 text-center">
-        {% asset logo-small.png alt="PyCon India 2019 Logo" width="70%" %}
+        {% asset logo-small.png alt="PyCon India 2019" width="70%" %}
```

Comment:
If we consider this just to be an eye candy, we set it to alt=""

Reply:
I don't think the footer can be considered as eye candy.